tests/subsys/suit/storage_nrf54h20/CMakeLists.txt

```diff
@@ -15,10 +15,12 @@ target_include_directories(app PRIVATE
   ${SUIT_SUBSYS_DIR}/storage/include
 )
 
-target_sources(app PRIVATE
-  # Use nRF54H20-specific implementation.
-  ${SUIT_SUBSYS_DIR}/storage/src/suit_storage_nrf54h20.c
-)
+if(CONFIG_BOARD_NATIVE_SIM)
```

Comment:
To my understanding - this test shall always use the nrf54h20-specific file.

Why do you need to add this `if`?

Reply:
According to changes introduced in https://github.com/nrfconnect/sdk-nrf/pull/20064, this test should no longer enable the `SUIT_STORAGE` symbol, include manually path to the headers and define the missing symbols, see: 
https://github.com/nrfconnect/sdk-nrf/blob/ab1330a0fb2d36f98d3365180df2d339ee503a93/tests/subsys/suit/orchestrator/orchestrator_sdfw_nrf54h20/CMakeLists.txt#L22

Reply:
This is build for the out of tree board which is nrf54h20 SoC, so the Kconfig symbol that it previously had to force select is now selected by default meaning that it doesn't need to include these files since these files are for native_sim only where it can't select the symbol due to it not being an nrf54h20 SoC

Reply:
I've prepared a fix for this here: https://github.com/nrfconnect/sdk-nrf/pull/20128

Reply:
ping @tomchy